src/main/groovy/de/dkfz/roddy/config/ResourceSet.groovy

```diff
@@ -94,7 +94,11 @@ class ResourceSet {
         return storage != null
     }
 
-    Duration getWalltime() {
+    TimeUnit getWalltime() {
```

Comment:
Unfortunately it's not that simple. The @deprecated constructor is not obsolete and should be used instead of the other one. Also the conversion code here is not valid (or at least not necessary), as our walltime object is of type TimeUnit!.
The Walltime2Duration makes sense, but would fail here, if you change the walltime field back to TimeUnit.